CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 
 ### App Center Distribute
 
+* **[Breaking change]** Add `setUpdateTrack` method to be able to explicitly set either `MSUpdateTrackPrivate` or `MSUpdateTrackPublic` update track. By default, a public distribution group will be used.
```

Comment:
Getter method is also provided, we need to mention it.

Reply:
Should also mention that we don't support multiple public groups anymore.